test/api/core_test.dart

```diff
@@ -460,7 +460,7 @@ void main() {
       });
     }
 
-    const packageInfo = PackageInfo(version: '0.0.1', buildNumber: '1');
+    const packageInfo = PackageInfo(version: '0.0.1', buildNumber: '1', packageName: 'com.zulip.flutter');
```

Comment:
Similarly, this can look like:
```suggestion
    const packageInfo = eg.packageInfo(version: '0.0.1', buildNumber: '1');
```

so that it doesn't have to make up a value for `packageName`, which this test doesn't care about. (And so that the reader therefore doesn't have to read that value and wonder if and how it affects the test after all.)